Cargo.lock

```diff
@@ -2960,7 +2960,7 @@ checksum = "49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f"
 
 [[package]]
 name = "vrl"
-version = "0.7.0"
+version = "0.8.0"
```

Comment:
Should this change have been in https://github.com/vectordotdev/vrl/pull/538?

Reply:
Yes, talked to @fuchsnj and we yanked the VRL version 0.8.0.